tests/integration/cattletest/core/test_allocation.py

```diff
@@ -1,7 +1,7 @@
 from common_fixtures import *  # NOQA
 
 
-def test_compute_free(admin_client, sim_context):
+def test_compute_free(admin_client, super_client, sim_context):
```

Comment:
This module (test_alloction.py) has a ton of usage of super_client where technically admin_client could be used. Specifically, it is used to create a lot of containers.

I presume the reasoning here is that we are not testing container creation/life cycle, we are testing backend (hidden) allocation logic and data structures. So, we ***don't care which client creates the containers*** because all we want to assert is that allocation is working correctly and that requires super_client. So, the tests are simpler if we just stick with a single client.

That accurate?